build-stanza.proj

```diff
@@ -1,5 +1,4 @@
 include "core/stanza.proj"
-include "core/debug/stanza.proj"
```

Comment:
Moved from `core/debug` to `debug` folder.

By default, the sources in `core` are not accessible by the end user because I don't want them to accidentally compile `core.stanza` when they should rely instead on the distributed `.pkg` files.